### PR TITLE
Give better names to Tournament and Random Groups

### PR DESCRIPTION
**Pull request by dnaranjo89**
The idea is to give a name the describes better what the draw does (i.e. *Tournaments* sounds like you are going to play something)

So the proposals are:
* Tournament --> Organise tournament
* Random groups  --> Build groups / Generate groups?

Additionally, this PR removes a translation since titles and help seemed to be translated twice

**Note: Translation for Random Groups is missing.** Someone with linux, run gettext :P

### CODE REVIEW

File: web/templates/index.html [hunk 1]
```diff
@@ -54,17 +54,17 @@
...
-            {% include "snippets/snippet_draw_box.html" with draw_type="groups" draw_icon="img/draw_icons/groups.png" title=_("Random Groups") help=_("Needs to group a bunch of people or items? Click here!") %}
+            {% include "snippets/snippet_draw_box.html" with draw_type="coin" draw_icon="img/draw_icons/coin.png" title="Flip a coin" help="Afraid of taking the wrong decision? Don't hesitate and throw the coin!" %}
+            {% include "snippets/snippet_draw_box.html" with draw_type="card" draw_icon="img/draw_icons/cards.png" title="Take a card" help="Let's play some drinking games!" %}
+            {% include "snippets/snippet_draw_box.html" with draw_type="dice" draw_icon="img/draw_icons/dice.png" title="Roll a dice" help="Don't you have a die? Just roll this one" %}
+            {% include "snippets/snippet_draw_box.html" with draw_type="tournament" draw_icon="img/draw_icons/tournament.png" title="Tournament" help="Organising a competition? This is your draw!" %}
+            {% include "snippets/snippet_draw_box.html" with draw_type="groups" draw_icon="img/draw_icons/groups.png" title="Random Groups" help="Needs to group a bunch of people or items? Click here!" %}
```
Comment (Mariocj89) [inline web/templates/index.html]: Typo in needs

**Comment by Mariocj89** [comment]
I'm OK about giving better names

**Comment by Mariocj89** [comment]
@dnaranjo89 Any update? Can we create an issue and close this PR?

**Comment by dnaranjo89** [comment]
@Mariocj89 We need to rename the draws, but I can not make the translations so make it?

**Pull request closed without merging**